metrics-operator/controllers/common/providers/datadog/datadog.go

```diff
@@ -0,0 +1,82 @@
+package datadog
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	//nolint:gci
```

Comment:
can we get rid of these `nolint` comments?

Reply:
updated